packages/storybook/.storybook/main.js

```diff
@@ -67,5 +55,9 @@ export function webpackFinal(config) {
 }
 export const staticDirs = ['public'];
 export const framework = {
-    name: '@storybook/html-webpack5'
+    name: getAbsolutePath('@storybook/html-vite')
 };
+
```

Comment:
The storybook auto migration tool re-added this. It had previously caused a build issue and [we removed it](https://github.com/ni/nimble/pull/2497), but I didn't run into any issues this time around.